app/views/layouts/application.html.haml

```diff
@@ -18,6 +18,9 @@
         .row.flash-messages
           = flash_messages
         .row.flash-messages
+          %noscript
+            .alert.alert-danger
+              JavaScript is deactivated in your browser. You might not be able to use all contents.
```

Comment:
1. Please use i18n
2. I'd rather say "Some features are not accessible without JavaScript." instead of "You might not be able to use all contents."